StreamChatCoreTests/Integration Tests/AttachmentTypeTests.swift

```diff
@@ -0,0 +1,51 @@
+import XCTest
+import StreamChatCore
+
+class AttachmentTypeTests: XCTestCase {
```

Comment:
Can you move the test file
from `StreamChatCoreTests/Integration Tests/AttachmentTypeTests.swift`
to `StreamChatCoreTests/Unit Tests/AttachmentTypeTests.swift `